scripts/setup-macos.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
If we're going to be pedantic, I'd suggest 
`!/usr/bin/env bash` instead of `/bin/bash`. I can't remember if the location of 'bash' is "guaranteed" to be in `/bin/`, unlike 'sh`, which I believe is required to be there by some POSIX-like standard.

Reply:
Nice catch! I'm surprised that it's not guaranteed on mac, but oh well :)